### PR TITLE
[Doc] Add Chinese doc for `waymo_det.md`

### PR DESCRIPTION
**Pull request by THU17cyz**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation
Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification
Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)
Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)
If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs_zh-CN/datasets/waymo_det.md [hunk 1]
```diff
@@ -1 +1,170 @@
...
+│   │   │   ├── waymo_gt_database
+│   │   │   ├── waymo_infos_trainval.pkl
+│   │   │   ├── waymo_infos_train.pkl
+│   │   │   ├── waymo_infos_val.pkl
+│   │   │   ├── waymo_infos_test.pkl
+│   │   │   ├── waymo_dbinfos_train.pkl
+
+```
+
+因为 Waymo 数据的来源包含数个相机，这里我们将每个相机对应的图像和标签文件分别存储，并将位姿文件存储下来以供后续处理连续多帧的点云。我们使用 `{a}{bbb}{ccc}` 的名称编码方式为每帧数据命名，其中 `a` 是不同数据拆分的前缀（`0` 指代训练集，`1` 指代验证集，`2` 指代测试集），`bbb` 是分割部分 (segment) 的索引，而 `ccc` 是帧索引。您可以轻而易举地按照如上命名规则定位到所需的帧。我们将训练和验证所需数据按 KITTI 的方式集合在一起，然后将训练集/验证集/测试集的索引存储在 `ImageSet` 下的文件中。
```
Comment (wHao-Wu) [inline docs_zh-CN/datasets/waymo_det.md:96]: 位姿文件 - > 位置文件？

File: docs_zh-CN/datasets/waymo_det.md [hunk 2]
```diff
@@ -1 +1,170 @@
...
+   sudo apt install build-essential
+
+   ./configure.sh
+   bazel clean
+
+   bazel build waymo_open_dataset/metrics/tools/compute_detection_metrics_main
+   cp bazel-bin/waymo_open_dataset/metrics/tools/compute_detection_metrics_main ../mmdetection3d/mmdet3d/core/evaluation/waymo_utils/
+   ```
+
+接下来，您就可以在 Waymo 上评估您的模型了。如下示例是使用 8 块 GPU 在 Waymo 上用 Waymo 评价指标评估 PointPillars 模型的情景：
```
Comment (wHao-Wu) [inline docs_zh-CN/datasets/waymo_det.md:123]: `GPU` -> `GPUs`

File: docs_zh-CN/datasets/waymo_det.md [hunk 3]
```diff
@@ -1 +1,170 @@
...
+
+1. 有时用 `bazel` 构建 `compute_detection_metrics_main` 的过程中会出现如下错误：`'round' 不是 'std' 的成员` (`'round' is not a member of 'std'`)。我们只需要移除该文件中，`round` 前的 `std::`。
+
+2. 考虑到 Waymo 上评估一次耗时不短，我们建议只在模型训练结束时进行评估。
+
+3. 为了在 CUDA 9 环境使用 TensorFlow，我们建议通过编译 TensorFlow 源码的方式使用。除了官方教程之外，您还可以参考该[链接](https://github.com/SmileTM/Tensorflow2.X-GPU-CUDA9.0)以寻找可能合适的预编译包以及编译源码的实用攻略。
+
+## 测试并提交到官方服务器
+
+如下是一个使用 8 块 GPU 在 Waymo 上测试 PointPillars，生成 bin 文件并提交结果到官方榜单的例子：
```
Comment (wHao-Wu) [inline docs_zh-CN/datasets/waymo_det.md:144]: `GPU` -> `GPUs`

File: docs_zh-CN/datasets/waymo_det.md (hunk 1 above)
Comment (THU17cyz) [reply docs_zh-CN/datasets/waymo_det.md:96]: pose 不是位姿嘛？

Comment (THU17cyz) [reply docs_zh-CN/datasets/waymo_det.md:96]: 改为了 `相机位姿 (pose) 文件`，你看如何

Comment (wHao-Wu) [reply docs_zh-CN/datasets/waymo_det.md]: 感觉应该可以

**Review by Tai-Wang — APPROVED** [review]

**Pull request merged**